cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/DefaultStore.java

```diff
@@ -954,6 +1026,10 @@ void clear() throws Exception {
     if (admin != null) {
       admin.truncate();
     }
+    admin = dsFramework.getAdmin(WORKFLOW_STATS_INSTANCE_ID, null);
+    if (admin != null) {
+      admin.truncate();
```

Comment:
we can reduce code duplication. 
``
clear(DatasetAdmin admin) {
...
}
``
and call 
``
clearAdmin(dsFramework.getAdmin(WORKFLOW_STATS_INSTANCE_ID, null)); 
clearAdmin(dsFramework.getAdmin(APP_META_INSTANCE_ID, null));
``